vars.prod.yml

```diff
@@ -1,4 +1,8 @@
 app_name: harvesting-logic
 database_name: harvesting-logic-db
 
-route-external: harvester-prod-datagov.app.cloud.gov
\ No newline at end of file
+route-external: harvester-prod-datagov.app.cloud.gov
+
+CLIENT_ID: urn:gov:gsa:openidconnect.profiles:sp:sso:gsa:datagov-prod-harvest-admin
+ISSUER: https://idp.int.identitysandbox.gov
```

Comment:
will need prod and staging values

Reply:
right, we will need make request to have it in prod and staging, for now we just host it in the sandbox.